### ISSUE TITLE
Bad version in installation docs

### ISSUE THREAD

**Issue by andreichirkunovgoon** [malcommac/SwiftDate#183]
let date2 = NSDate(year: 2015, month: 12, day: 25, hour: 14) 

gives an error. I installed using pods. Is this issue known to you? The other example, 

let date1 = "2015-01-05T22:10:55.200Z".toDate(DateFormat.ISO8601)

works well. Thank you in advance for your reply!

**Comment by Hout** [comment]
```
let date2 = NSDate(year: 2015, month: 12, day: 25, hour: 14) 
print(date2)
```
Shows: `2015-12-25 13:00:00 +0000`. (Please note that it runs in the CET time zone, so this is correct)

I.e. unable to reproduce...

**Comment by Hout** [comment]
Which version of the pod do you use?

**Comment by andreichirkunovgoon** [comment]
I used v 2.0.2, which was specified in the installation guide. I didn't check for newer versions. I now updated to v3 and it works. Thank you for your hint! You did amazing code!!

**Comment by Hout** [comment]
Thx for the feedback.

I will correct the version in the installation guide.

**Comment by Hout** [comment]
Fixed in 3.0.9
